components/zcash_protocol/src/local_consensus.rs

```diff
@@ -43,6 +43,37 @@ pub struct LocalNetwork {
     pub z_future: Option<BlockHeight>,
 }
 
+impl LocalNetwork {
+    pub fn new(
```

Comment:
- Document the constructor.
- The arguments should be checked for ordering. It is fine to activate two sequential network upgrades at the same height, but it is not okay to change the order in which NUs activate. Either make the constructor family, or just document that it panics (which is probably fine for something only used for development).